tests/integration/dumpMetadata/helpers/requestSender.ts

```diff
@@ -0,0 +1,28 @@
+import * as supertest from 'supertest';
+import { Application } from 'express';
+
+import { container } from 'tsyringe';
+import { ServerBuilder } from '../../../../src/serverBuilder';
+import { DumpMetadataFilter } from '../../../../src/dumpMetadata/models/dumpMetadataFilter';
+
+export function getApp(): Application {
+  const builder = container.resolve<ServerBuilder>(ServerBuilder);
+  return builder.build();
+}
+
+export function getMockedRepoApp(repo: unknown): Application {
+  container.register('DumpMetadataRepository', { useValue: repo });
+  return getApp();
+}
+
+export async function getDumpsMetadataByFilter(
+  app: Application,
+  filter: DumpMetadataFilter | Record<string, never>,
+  responseType = 'application/json'
```

Comment:
You never this parameter, so its fine to just hardcode it.

Reply:
done.